docs/custom_tool_guides/tool_guide.md

```diff
@@ -25,115 +25,48 @@ There are three types of tools:
 
 ## Step 3: Implement the Tool
 
-Add your tool implementation [here](https://github.com/cohere-ai/toolkit/tree/main/src/community/tools) (please note that this link might change). The specific subfolder used will depend on the type of tool you're implementing.
+Add your tool implementation [here](https://github.com/cohere-ai/toolkit/tree/main/src/community/tools) (please note that this link is subject to change).
 
-If you need to install a new module to run your tool, execute the following command and run `make dev` again.
+If you need to install a new library to run your tool, execute the following command and run `make dev` again.
 
 ```bash
 poetry add <MODULE> --group community
 ```
+### Implementing a Tool
 
-If you're working on a File or Data Loader, follow the steps outlined in [Implementing a Retriever](#implementing-a-retriever).
+Add the implementation inside a tool class that inherits from `BaseTool`. This class will need to implement
```

Comment:
Should we mention that the UI uses the `text` field to populate the citations? I've seen people implementing tools that didn't return `text` confused with the UI

Reply:
Good point

Reply:
@lusmoura done